scripts/install.sh

```diff
@@ -29,7 +29,12 @@ then
 fi
 
 LUID="LOCAL_UID=`id -u $USER`"
-LGID="LOCAL_GID=`getent group docker | cut -d: -f3`"
+if [ "$OS" == "lin" -a `id -u $USER` -eq 0 ]
+then
+    LGID="LOCAL_GID=`getent group docker | cut -d: -f3`"
```

Comment:
Should we get rid of this condition now based on conversations from yesterday?

Reply:
Yes I'm simplifying all this part.
Unfortunately I was stuck on other things right after our conversations, but now I'm on it 👍